web/lib/supabase/bets.ts

```diff
@@ -20,3 +20,19 @@ export async function getOlderBets(
 
   return data.map((d: JsonData<Bet>) => d.data)
 }
+
+export async function getTotalBetCount(contractId: string) {
+  const { count } = await run(
+    db
+      .from('contract_bets')
+      .select('*', { count: 'exact', head: true })
+      .eq('contract_id', contractId)
+      .eq('data->isChallenge', false)
+      .eq('data->isRedemption', false)
+      .eq('data->isAnte', false)
+  )
+
+  // should never happen
+  if (count === null) throw new Error("Couldn't fetch bet count from database")
+  return count
```

Comment:
I agree this condition makes no sense so why are you checking it? Seems like you should just cast `count` to `number`.

(P.S. I am definitely going to work on making this stuff have better Typescript support)

Reply:
Sure, casting works fine too.